.github/workflows/build-tasks.yml

```diff
@@ -1,21 +1,68 @@
 name: build-tasks
+
 on:
   # this is meant to be run on an approved PR branch for convenience
   workflow_dispatch:
+
+# We derive a unique ID here to make sure that we don't get into a situation
+# where different runners pick different tags (eg: for builds near midnight, or
+# in different time zones).  The manifest job picks the final name.
+env:
+  tag: quay.io/cockpit/tasks
+  tmptag: quay.io/cockpit/tasks-tmp:${{ github.run_id }}.${{ github.run_attempt }}
+
 jobs:
   build:
-    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        build:
+          - label: amd64
+            runner: buildjet-2vcpu-ubuntu-2204
+          - label: arm64
+            runner: buildjet-2vcpu-ubuntu-2204-arm
```

Comment:
I'm surprised that we don't need any secret/token for this? They just bill it to the org?

Reply:
No.  I signed us up and we get an initial 5$ credit.  We need to add a credit card once that runs out.

Reply:
I wasn't totally clear: as part of the signup, I added their app to our org.